### PR TITLE
Remove unused relationships code (including relationships custom field)

### PR DESCRIPTION
**Pull request by brmscheiner**

### CODE REVIEW

File: src/constants/fieldTypes.js [hunk 1]
```diff
@@ -85,22 +84,6 @@ export const fieldTypeChoices = [
...
-  {
-    labelId: 'RELATIONSHIPS_SELECTOR',
-    value: fieldTypes.relationships,
-    defaultValue: [],
-    backendType: backendTypes.string,
-    backendMultiple: true,
-    configuration: [
-      {
-        labelId: 'RELATIONSHIP_OPTIONS',
-        descriptionId: 'RELATIONSHIP_OPTIONS_HELPER_TEXT',
```
Comment (Emily-Ke) [inline src/constants/fieldTypes.js:97]: Can these also be removed from `en.json`?

File: src/pages/socialGroups/SocialGroups.jsx [hunk 2]
```diff
@@ -44,18 +44,21 @@ export default function SocialGroups() {
...
+              key="view"
+              variant="view"
+              href={`/social-groups/${guid}`}
+            />
+            <ActionIcon
+              key="delete"
+              variant="delete"
+              onClick={() => setPendingDeleteGuid(guid)}
+            />
+            ,
```
Comment (Emily-Ke) [inline src/pages/socialGroups/SocialGroups.jsx:59]: This comma should be removed.

File: src/components/inputs/inputMap.js [hunk 3]
```diff
@@ -32,7 +31,6 @@ const inputMap = {
   [fieldTypes.date]: DateInput,
   [fieldTypes.daterange]: DateRangeInput,
   [fieldTypes.feetmeters]: FeetMetersInput,
-  [fieldTypes.relationships]: RelationshipsInput,
```
Comment (Emily-Ke) [inline src/components/inputs/inputMap.js:35]: Can the `RelationshipsInput` file also be deleted?

File: src/pages/fieldManagement/settings/FieldTypeSelector.jsx [hunk 4]
```diff
@@ -31,7 +31,6 @@ const fieldTypeCategories = [
       fieldTypes.file,
       fieldTypes.individual,
       fieldTypes.latlong,
-      fieldTypes.relationships,
```
Comment (Emily-Ke) [inline src/pages/fieldManagement/settings/FieldTypeSelector.jsx:34]: I don't see where the `FieldTypeSelector` is used. Can the entire file be deleted?

File: src/constants/fieldTypes.js (hunk 1 above)
Comment (brmscheiner) [reply src/constants/fieldTypes.js:97]: looks like they can!

File: src/components/inputs/inputMap.js (hunk 3 above)
Comment (brmscheiner) [reply src/components/inputs/inputMap.js:35]: yes

File: src/pages/fieldManagement/settings/FieldTypeSelector.jsx (hunk 4 above)
Comment (brmscheiner) [reply src/pages/fieldManagement/settings/FieldTypeSelector.jsx:34]: yes

**Review by Emily-Ke — APPROVED** [review]
Looks good to me.

**Pull request merged**